src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -355,6 +440,25 @@ def heat(scenario):
             ),
         }
 
+        # Insert lifetime
+        parameters["lifetime"] = {
+            "central_water_tank": read_costs(
+                costs, "central water tank storage", "lifetime"
+            ),
+            "rural_water_tank": read_costs(
+                costs, "decentral water tank storage", "lifetime"
+            ),
+        }
+
+        # Insert annualized capital costs
+        parameters["capital_cost"] = {}
+
+        for comp in parameters["overnight_cost"].keys():
+            parameters["capital_cost"][comp] = annualize_capital_costs(
```

Comment:
so for there is no annualization for the mobility part. Would it make sense to at least add a comment or so that once that is added the same function is used?

Reply:
Done in [166bae3](https://github.com/openego/eGon-data/pull/681/commits/166bae32eee9326253c70eac5ec992f04ab708dd)